cli-output/src/cli_output.rs

```diff
@@ -1528,6 +1528,100 @@ impl fmt::Display for CliProgramAuthority {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CliUpgradeableProgram {
+    pub program_id: String,
+    pub program_executable: bool,
+    pub program_lamports: u64,
+    pub programdata_address: String,
+    pub programdata_lamports: u64,
+    pub programdata_authority: String,
+    pub programdata_slot: u64,
+    pub programdata_data_len: usize,
+    pub programdata_program_len: usize,
```

Comment:
```suggestion
    pub executable: bool,
    pub lamports: u64,
    pub program_data_address: String,
    pub program_data_lamports: u64,
    pub program_data_authority: String,
    pub program_data_slot: u64,
    pub program_data_len: usize,
    pub program_len: usize,
```

Reply:
These work a lot better in camel-case json

Reply:
I wonder if `program_data_slot` ought to be more descriptive. `last_upgrade_slot`? (does it need the program_data prefix?)